gelato-relay/src/lib.rs

```diff
@@ -0,0 +1,122 @@
+mod types;
+pub use types::*;
+
+use color_eyre::eyre::Result;
+use std::collections::HashMap;
+
+const DEFAULT_URL: &str = "https://relay.gelato.digital";
+
+#[derive(Debug, Clone)]
+pub struct GelatoClient {
+    url: String,
+}
+
+impl Default for GelatoClient {
+    fn default() -> Self {
+        Self::new(DEFAULT_URL.to_owned())
+    }
+}
+
+impl GelatoClient {
+    pub fn new(url: String) -> Self {
+        Self { url }
+    }
+
+    pub async fn send_relay_transaction(
+        &self,
+        chain_id: usize,
+        dest: &str,
+        data: &str,
+        token: &str,
+        relayer_fee: usize,
+    ) -> Result<RelayResponse, reqwest::Error> {
+        let params = RelayRequest {
+            dest: dest.to_owned(),
+            data: data.to_owned(),
+            token: token.to_owned(),
+            relayer_fee: relayer_fee.to_string(),
+        };
+
+        let url = format!("{}/relays/{}", &self.url, chain_id);
+        let res = reqwest::Client::new()
+            .post(url)
+            .json(&params)
+            .send()
+            .await?;
+
+        res.json().await
+    }
+
+    pub async fn send_forward_request(
+        &self,
+        params: &ForwardRequest,
+    ) -> Result<RelayResponse, reqwest::Error> {
+        let url = format!(
+            "https://gateway.api.gelato.digital/metabox-relays/{}",
```

Comment:
Should use `self.url` but we're using staging endpoints atm. Will change this `self.url` once Gelato has finalized our testnets under prod endpoint

Reply:
Are you referring to all hardcoded URLs here? Seems like they should live somewhere else or at least be in `const` form

Reply:
Yeah was just hardcoding temporarily while using their staging endpoints. Switched to prod endpoint see [this commit](https://github.com/nomad-xyz/rust/pull/178/commits/7ea4be07c8bfc20e80b760b122bf629b69555e6c)